src/main/java/Remy/command/FindCommand.java

```diff
@@ -0,0 +1,42 @@
+package Remy.command;
+import Remy.ChatbotException;
+import Remy.Storage;
+import Remy.Task.TaskList;
+import Remy.Ui;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A Command that finds matching Tasks upon executing.
+ */
+public class FindCommand extends Command {
+    private String searchString;
+    public static final String COMMAND_WORD = "find";
+
+    // Regex pattern for "find {keyword}" format
+    private static final Pattern FIND_PATTERN = Pattern.compile("^find\\s+(\\S+)$", Pattern.CASE_INSENSITIVE);
+
+    public FindCommand(String input) throws ChatbotException {
+        Matcher matcher = FIND_PATTERN.matcher(input.trim());
+        if (matcher.matches()) {
+            this.searchString = matcher.group(1).toLowerCase();
+        } else {
+            throw new ChatbotException("format wrong bro");
```

Comment:
Perhaps on top of telling "bro" that his format is wrong, you could additionally provide the correct format to him so "bro" knows what he is doing wrong. E.g find ___ -> Did you forget to input a word?